packages/bot/src/transformers/types.ts

```diff
@@ -1824,3 +1824,27 @@ export interface SoundboardSound {
   /** The user who created this sound */
   user?: User
 }
+
+/** https://discord.com/developers/docs/resources/lobby#lobby-object-lobby-structure */
+export interface Lobby {
+  /** The id of this channel */
+  id: bigint
+  /** application that created the lobby */
+  applicationId: bigint
+  /** dictionary of string key/value pairs. The max total length is 1000. */
+  metadata?: Record<string, string>
+  /** members of the lobby */
+  members: LobbyMember[]
+  /** the guild channel linked to the lobby */
+  linkedChannel?: Channel
+}
+
+/** https://discord.com/developers/docs/resources/lobby#lobby-member-object-lobby-member-structure */
+export interface LobbyMember {
+  /** The id of the user */
+  id: bigint
+  /** dictionary of string key/value pairs. The max total length is 1000. */
+  metadata?: Record<string, string>
+  /** lobby member flags combined as as bitfield */
+  flags?: number
```

Comment:
https://discord.com/developers/docs/resources/lobby#lobby-member-object-lobby-member-flags

Should we also make a bitfield toggle for this flag to be consistent with other stuff that also use bitfield toggles?